README.md

```diff
@@ -22,14 +22,14 @@ preview environment's `API` and `Search API` boxes, or use local API instances i
 you have them running:
 
 ```
-export DM_API_URL=http://localhost:5000
-export DM_BUYER_FRONTEND_API_AUTH_TOKEN=<auth_token_accepted_by_api>
+export DM_DATA_API_URL=http://localhost:5000
```

Comment:
Have these environment variable name changes been made in the AWS setup already? Or do they need a change there too?